### DISCUSSION TITLE
IIW Fall 2023 Discussion Topics

### DISCUSSION THREAD

**Discussion by tkuhrt** [openwallet-foundation/architecture-sig#53] [General]
What discussion topics would you like to see on the agenda for IIW Fall 2023?

**Comment by tkuhrt** [comment]
App Attestation

**Comment by tkuhrt** [comment]
Universal Wallet Backup Containers - some discussion has already happened here: https://github.com/decentralized-identity/universal-wallet-backup-containers

**Comment by tkuhrt** [comment]
What components do you want to see brought to OWF

**Comment by tkuhrt** [comment]
Human Binding to Wallets (Biometrics) - Examples

**Comment by tkuhrt** [comment]
Universal verification component

**Comment by tkuhrt** [comment]
Wallet Selector

**Comment by kayaelle** [comment]
Education & Workforce Wallet Requirements/Needs (with VC-EDU)
